clients/pricing-client/package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "test": "jest",
     "bundle-definition": "webpack",
-    "openapi": "node ../../scripts/update-openapi.js /Users/paulohenriques/Documents/pricing-api/lambda/ApiHandlerFunction/openapi.yml",
+    "openapi": "node ../../scripts/update-openapi.js /Users/juanjoseguevara/Desktop/code/pricing-api/lambda/ApiHandlerFunction/openapi.yml",
```

Comment:
Shouldn't this be a relative path?

Reply:
https://docs.api.epilot.io/pricing-api.yaml

This one should be the correct one

Reply:
yeah, maybe that would be best, but this openapi.yml file exists on another project (pricing-api), so it can't be written reliably as a relative path. I'm unsure what would be better, to move the file over this project(sdk-js) or to move the script over pricing-api, or to keep it as it is. @paulofph what are your thoughts?